SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -460,6 +505,26 @@ void USpatialSender::SendComponentUpdates(UObject* Object, const FClassInfo& Inf
 
 	TOptional<Trace_SpanId> CauseSpanId = EventTracer->PopLatentPropertyUpdateSpanId(Object);
 
+	if (EventTracer && CauseSpanId.IsSet() && RepChanges->RepChanged.Num() > 0) // Only need to add these if they are actively being traced
+	{
+		TArray<Trace_SpanId> IndividualPropertySpans;
+		for (FChangeListPropertyIterator Itr(RepChanges); Itr; ++Itr)
+		{
+			UProperty* Property = *Itr;
+			TOptional<Trace_SpanId> LinearTraceSpan =
+				CauseSpanId.IsSet() ? EventTracer->CreateSpan(&CauseSpanId.GetValue(), 1) : EventTracer->CreateSpan();
+			EventTracer->TraceEvent(
+				FSpatialTraceEventBuilder::CreateSendPropertyLinearTraceEvent(EventTraceUniqueId::GenerateForProperty(EntityId, Property)),
```

Comment:
Not necessary but if you fancy it, it might be nice to move the existing FSpatialTraceEventBuilder::CreateSendPropertyUpdates here and do it per property and give it a property name. Might be nice to see the names of the properties being sent.